source/docs/dashboard.md

```diff
@@ -0,0 +1,30 @@
+title: Using the dashboard
+---
+
+Embark provides a very useful dashboard view that aims to make it easier for developers to keep track of things, such as running services, compilation and deployments. In this guide we'll take a closer look at the dashboard and its features.
+
+## Overview
+
+Embark spins up the dashboard view automatically for us, whenever we run `embark run` inside an Embark project. Here's what it looks like:
+
+![Dashboard](/assets/images/embark-dashboard.png)
+
+The dashboard is separated into different sections, each with their own responsibility. The sections are:
+
+- **Contracts** - This section shows you not only all the Smart Contracts within your project, but also their status on whether they are being deployed at the moment or not. Notice that, as we make chances to our Smart Contracts, the dashboard will reflect the automatic redeployment in this section as well, making it very easy to stay on top of things.
+  <br>
```

Comment:
Yes and no. What I want to achieve as a newline after the list point, but with vanilla markdown, this is not possible. If we introduce two empty lines, this will actually result in one `<ul>` per `<li>` which is semantically incorrect. That's why I went with `<br>` for now.

Obviously, this should actually be tackled using CSS

Reply:
Fixed this with CSS now